Modix.Data/Models/Core/MessageEntity.cs

```diff
@@ -16,44 +17,42 @@ internal class MessageEntity
         [Required]
         public ulong ChannelId { get; set; }
 
+        public virtual GuildChannelEntity Channel { get; set; }
```

Comment:
We don't ACTUALLY plan on doing lazy-loading, do we?

Reply:
No, but I just wanted to keep it consistent. IMO we can just drop `virtual` everywhere, really.